kinase-bioactivities-in-chembl/kinase-bioactivities-in-chembl.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<p>That's probably chembl version 28. Could we bring in here the <code>CHEMBL_VERSION</code> variable to adjust the number with every release?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openkinome/kinodata/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>